### DISCUSSION TITLE
Problemi nella connessione da remoto

### DISCUSSION THREAD

**Discussion by baert3nder** [indomus/forum#2998] [HUB: Home Assistant]
Buongiorno, ho un problema nel connettermi da remoto all'HUB di Home Assistant: quando digito ``` https://xxxxxxx.duckdns.org:8123```  sul pc mi compare questo messaggio: “ERR_ADDRESS_UNREACHABLE”.
L’unico modo che ho per connettermi è scrivere direttamente l’indirizzo IP dell’hub ma mi compare l’avvertimento di sito non sicuro.
Ho Home Assistant OS su un miniPC (Mini PC ACE NiPoGi AK1 Plus)  e ho seguito questa guida: https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-installato-con-distribuzione-hassio/. Ho fatto tutti i passaggi descritti(dalla registrazione su duckDNS fino al port forwarding della porta sul terminale) senza nessun problema, l’unico passaggio che ho saltato è stata l’installazione di Let’s Encrypt ma nella nota c’è scritto che non è necessario. Se qualcuno ha avuto lo stesso problema oppure c’è stato un'altro thread in cui lo stesso problema apprezzerei la collaborazione.
(È la prima volta che scrivo nel forum quindi accetterò ogni critica che mi sarà fatta) 
Grazie

**Comment by snakuzzo** [comment]
Se stai usando l'addon duckdns, non ti serve quello di let's encrypt, questo perché ci pensa direttamente l'addon di duckdns a superare le challenge per farsi rilasciare il certificato da let's encrypt.
Quanto al tuo errore, vuoi dire che l'indirizzo ip associato al tuo FQDN (il dominio registrato su duckdns e configurato sull'addon) non risolve con il tuo ip pubblico. Detto ciò...sei sicuro di avere un indirizzo ip pubblico? Che gestore hai?

**Reply by baert3nder** [reply]
Ciao, io ho WindTre come gestore e ho appena verificato su "What Is My IP Address" che il mio IP è statico.

**Reply by snakuzzo** [reply]
Non ti ho chiesto se fosse statico, ma pubblico

**Reply by baert3nder** [reply]
Scusa mi sono confuso, è pubblico (è un po difficile da reperire come informazione per i neofiti)

**Reply by baert3nder** [reply]
Aggiornamento: ho rifatto tutta la procedura saltando la parte dove aggiorno il record su DuckDNS tramite automazione Home Assistant (https://indomus.it/guide/aggiornare-il-proprio-record-su-duckdns-tramite-automazione-home-assistant/), quando avvio DuckDNS e guardo il registro mi dà questo
```
s6-rc:` info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
# INFO: Using main config file /data/workdir/config
+ Account already registered!
[14:47:12] INFO: OK
111.222.333.444 (IP pubblico del router)

NOCHANGE
[14:47:12] INFO: Renew certificate for domains: xxx.duckdns.org and aliases: 
# INFO: Using main config file /data/workdir/config
Processing xxx.duckdns.org
 + Signing domains...
 + Generating private key...
 + Generating signing request...
 + Requesting new certificate order from CA...
 + Received 1 authorizations URLs from the CA
 + Handling authorization for xxx.duckdns.org
 + 1 pending challenge(s)
 + Deploying challenge tokens...
OK + Responding to challenge for xxx.duckdns.org authorization...
 + Cleaning challenge tokens...
OK + Challenge validation has failed :(
ERROR: Challenge is invalid! (returned: invalid) (result: ["type"]	"dns-01"
["status"]	"invalid"
["error","type"]	"urn:ietf:params:acme:error:dns"
["error","detail"]	"DNS problem: SERVFAIL looking up TXT for _acme-challenge.xxx.duckdns.org - the domain's nameservers may be malfunctioning"
["error","status"]	400
["error"]	{"type":"urn:ietf:params:acme:error:dns","detail":"DNS problem: SERVFAIL looking up TXT for _acme-challenge.xxx.duckdns.org - the domain's nameservers may be malfunctioning","status":400}
["url"]	"https://acme-v02.api.letsencrypt.org/acme/chall-v3/330928235717/YPyWVg"
["token"]	"XXXXXXXXXXXXXXXXXXXXXXX"
["validated"]	"2024-03-26T13:47:18Z")
```
spero che con un log si possa capire meglio qual è il problema
